bookStore.js

```diff
@@ -0,0 +1,41 @@
+let books = JSON.parse(localStorage.getItem('books')) || [];
+
+function displayBooks() {
+  const bookList = document.getElementById('bookList');
+  bookList.innerHTML = '';
+  // Define the removeBook function before it is called
+  function removeBook(index) {
+    books = books.filter((book, i) => i !== index);
+    localStorage.setItem('books', JSON.stringify(books));
+    displayBooks();
+  }
+  books.forEach((book, index) => {
+    const li = document.createElement('li');
+    li.innerHTML = `
+    <p>${book.title}</p>
+    <p>${book.author}</p>
+    <hr>
+    `;
+    const removeBtn = document.createElement('button');
+    removeBtn.textContent = 'Remove';
+    removeBtn.addEventListener('click', () => {
+      removeBook(index);
+    });
+    li.appendChild(removeBtn);
+    bookList.appendChild(li);
+  });
+}
+
+function addBook(event) {
+  event.preventDefault();
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  books.push({ title, author });
```

Comment:
- [OPTIONAL] I highly recommend giving each newly added book the unique `id` which may help in filtering books easily.
- [OPTIONL] I highly recommend you do not use the `push` method which may mutate the array instead you can use the [Destructing assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) method. You can check [more here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) on how to use it.